include/boost/geometry/algorithms/convert.hpp

```diff
@@ -175,11 +164,16 @@ struct range_to_range
     {
         geometry::clear(destination);
 
-        rview_type rview(source);
+        using view_type = detail::closed_clockwise_view
+            <
+                Range1 const,
+                geometry::closure<Range1>::value,
+                Reverse ? counterclockwise : clockwise
+            >;
```

Comment:
Clear, I'm glad with this.
One question, isn't it sufficient to just pass `Reverse` here? Do we need the ternary?
I'm fine either way, you can decide.

Reply:
OK, I just reviewed the new definition of `closed_clockwise_view`. It makes sense like this, no need to change.
And I now understand better that if we pass `Reverse`, it should be specified here; if not, it's retrieved automatically.

But sometimes we need `Reverse` as a separate template parameter (for example for `difference` and therefore in general for all overlay), because we walk those geometries in opposite direction, on purpose.

Reply:
Yes, this is why the Reverse to Order (and previously Reverse to Direction) translation is here.

Another possibility would be to pass the Order instead of Reverse but I'd have to check all of the places to see if this is more clear.